HVZ.Persistence/Models/Game.cs

```diff
@@ -97,6 +108,8 @@ public Game(string name, string gameid, string creatorid, string orgid, Instant
         DefaultRole = defaultrole;
         Players = players;
         EventLog = eventLog;
+        OzPool = new List<string>();
```

Comment:
this should come from an argument, not be hardcoded to always be a new empty list.